gnovm/stdlibs/math/overflow/overflow.gno

```diff
@@ -2,8 +2,10 @@
 // NOTE: there was a bug with the original Quotient* functions, and
 // testing method. These have been fixed here, and tests ported to
 // tests/files/maths_int*.go respectively.
-// TODO: make PR upstream.
-package maths
+// Note: copied over from p/demo/maths & deprecated over there.
+package overflow
+
+import "math"
 
 /*
 Package overflow offers overflow-checked integer arithmetic operations
```

Comment:
This should go as a comment before the `package` clause, so it shows up in gno doc

Reply:
[d8fd1f5](https://github.com/gnolang/gno/pull/1698/commits/d8fd1f58bd95f4a02b4cc55c82f675fcbe22e4a9)